tools/inventory-downloader/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.20.12 AS build
+
+WORKDIR /src
+
+COPY src/go.mod ./
+COPY src/go.sum ./
+
+RUN go mod download
+
+COPY src ./
+
+RUN go build -o /inventory-downloader .
+
+FROM ubuntu AS bin
+
+COPY eu-central-1-bundle.crt /usr/local/share/ca-certificates/
```

Comment:
I believe we don't need this one - it's only necessary if we're talking to AWS RDS.

Reply:
Removed